### PR TITLE
9 cli tool fam

### PR DESCRIPTION
**Pull request by Fr3dMer**

### CODE REVIEW

File: PannelAppDB.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+
+
+
+def main():
+
+    # CLI 
+    cli = cli_module.cli_obj(sys.argv[1:])
+
+    #print(cli.args.panel_id)
```
Comment (AmyGrimwood) [inline PannelAppDB.py:20]: Should this comment be deleted

File: src/JSON_parsing.py
Comment (AmyGrimwood) [inline src/JSON_parsing.py:1]: Delete this script (old version of script)

File: PannelAppDB.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        print(query_version)
+        
+    else:
+        # No internet so get pannel_id from db and json data 
+        raise SystemExit("Could not connect to internet")
+
+    # Make a call to API and get GMS version 
+    gms_panel = api.get_gms_pannel(cli.args.panel_id)
+    #gms_panel = 4.3
+    #gms_pannel_version = parsing.get_version(gms_pannel)
```
Comment (AmyGrimwood) [inline PannelAppDB.py:43]: This shouldnt be commented

File: PannelAppDB.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        query_version = float(raw_data.get("version",None))
+        print(query_version)
+        
+    else:
+        # No internet so get pannel_id from db and json data 
+        raise SystemExit("Could not connect to internet")
+
+    # Make a call to API and get GMS version 
+    gms_panel = api.get_gms_pannel(cli.args.panel_id)
+    #gms_panel = 4.3
```
Comment (AmyGrimwood) [inline PannelAppDB.py:42]: Should this comment be removed?

**Review by AmyGrimwood — CHANGES_REQUESTED** [review]

**Review by AbiHaddon — APPROVED** [review]
Happy

**Review by AmyGrimwood — APPROVED** [review]
Changes made

**Pull request merged**